docker-entrypoint.sh

```diff
@@ -183,7 +183,7 @@ fi
 ors_engine_graphs_root_path=$(env | grep "^ors\.engine\.graphs_root_path=" | awk -F '=' '{print $2}')
 # get ors.engine.elevation.cache_path
 ors_engine_elevation_cache_path=$(env | grep "^ors\.engine\.elevation\.cache_path=" | awk -F '=' '{print $2}')
-# get ors.engine.source_file
+# get ors.engine.profile_default.build.source_file
 ors_engine_source_file=$(env | grep "^ors\.engine\.source_file=" | awk -F '=' '{print $2}')
```

Comment:
if the comment changes, shouldn't the code change as well?

Reply:
Yep. That was some halfway through fixing. The PR is not ready yet. I will do the docker fix now.